### PR TITLE
Update copy for '/financial-services'

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates copy as per [copydoc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit#)
- Updates logo sections patterns to use the current vanilla pattern
- Some formatting changes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10913.demos.haus/financial-services
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against copydoc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10905

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/financial-services/index.html [hunk 1]
```diff
@@ -236,6 +246,25 @@ <h2>Reduce IT operations cost</h2>
   </div>
 </section>
 
+<div class="u-fixed-width">
+  <hr class="p-separator"/>
+</div>
+
+<section class="p-strip">
```
Comment (minkyngkm) [inline templates/financial-services/index.html:253]: just wondering if this part starts with a new `section`. 

1. If it's a new section maybe we don't need a separator above and change the class here with `p-strip--light`? 
```suggestion
<section class="p-strip--light">
```
2. If it's part of the section above, remove this `section` tag?  

What do you think?

Comment (petesfrench) [reply templates/financial-services/index.html:253]: Great idea, I have removed the break and added --light. It looks a lot better now, thanks!

**Review by minkyngkm — COMMENTED** [review]
Looking good Pete! just wondering if `Secure and compliant`, `Redefine customer experience`, `Regain focus on business priorities`, `Enhance business resilience`, `Intelligent finance` and `Increase developer productivity` need to be in one `section` just like the section above.

File: templates/financial-services/index.html [hunk 2]
```diff
@@ -532,37 +581,39 @@ <h2>Intelligent finance</h2>
         <a href="/kubeflow">machine learning processes with Kubeflow</a>. With  Ubuntu, you benefit from perfect multi-cloud portability of AI/ML workloads.</p>
     </div>
     <div class="col-4 u-align--center u-hide--small">
-      <ul class="p-inline-images u-no-margin">
-        <li class="p-inline-images__item">
+      <div class="p-logo-section u-no-margin">
```
Comment (minkyngkm) [inline templates/financial-services/index.html:584]: Could we make the logos be centrally aligned or make two logos in one line? What do you think?

Comment (petesfrench) [reply templates/financial-services/index.html:576]: For sure! I have made these changes now

**Review by minkyngkm — APPROVED** [review]
LGTM! 👍

**Pull request merged**